src/ModifyTable.jl

```diff
@@ -7,29 +7,23 @@ Replace the parameter `var` with `expr` in the class type `class`.
 ```jldoctest
 julia> g=genchartab("GL2");
 
-julia> printval(g,class=1)
-Value of character type 1 on class type
-  1: exp(2π𝑖((2*i*k)//(q - 1)))
-Value of character type 2 on class type
-  1: q*exp(2π𝑖((2*i*k)//(q - 1)))
-Value of character type 3 on class type
-  1: (q + 1)*exp(2π𝑖((i*l + i*k)//(q - 1)))
-Value of character type 4 on class type
-  1: (q - 1)*exp(2π𝑖((i*k)//(q - 1)))
+julia> [g[i,1] for i in 1:4]
```

Comment:
```suggestion
julia> [c[1] for c in g]
```

Maybe this is a bit cleaner.

Reply:
Nice, I didn't realize we support this now.